packages/charts/react-charting/src/components/LineChart/LineChart.base.tsx

```diff
@@ -158,12 +158,17 @@ export interface ILineChartState extends IBasestate {
   nearestCircleToHighlight: ILineChartDataPoint | null;
 
   activeLine: number | null;
+
+  YValue: string | number;
+
+  legendVal: string;
```

Comment:
could you confirm if render callout per datapoint and render callout per stack are working as expected after your change.